### PR TITLE
Add about_CimSession.md.

### PR DESCRIPTION
**Pull request by sethvs**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [x] Impacts 6.0 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### CODE REVIEW

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md) | :warning:Warning | | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/4.0/Microsoft.PowerShell.Core/About/about_CimSession.md) | :warning:Warning | | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/5.0/Microsoft.PowerShell.Core/About/about_CimSession.md) | :warning:Warning | | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/5.1/Microsoft.PowerShell.Core/About/about_CimSession.md) | :warning:Warning | | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md)
[reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/4.0/Microsoft.PowerShell.Core/About/about_CimSession.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/5.0/Microsoft.PowerShell.Core/About/about_CimSession.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/5.1/Microsoft.PowerShell.Core/About/about_CimSession.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C28%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201804280953560557-2373%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cyL3ExFZTImMdYjDPh2vr15d8SZ0Ty0o%2FU2%2F0zpQu4I%3D&st=2018-04-28T09%3A59%3A00Z&se=2018-05-29T10%3A04%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sdwheeler** [comment]
@SteveL-MSFT Please review

File: reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+version of the Windows operating system.
+
+In contrast, a cmdlet run by using a PSSession runs on the target
+computer. It requires Windows PowerShell on the target system.
+Furthermore, the cmdlet sends data back to the local computer.
+Windows PowerShell manages the data sent over the connection, and keeps
+the size within the limits set by Windows Remote Management (WinRM). CIM
+sessions do not impose the WinRM limits.
+
+PSSessions only work with WinRM. CimSessions can use DCOM.
```
Comment (SteveL-MSFT) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md]: PSSessions also work with SSH (with PSCore6), PowerShell Direct (VMBus to containers/VMs), named pipes locally.  Might be better to exclude this line.

File: reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+for the connection, session ID, and instance ID.
+
+After you create a CimSession object that specifies information required to
+establish a connection, Windows PowerShell does not establish the
+connection immediately. When a cmdlet uses the CIM session,
+Windows PowerShell connects to the specified computer, and then, when the
+cmdlet finishes, Windows PowerShell terminates the connection.
+
+If you create a PSSession instead of using a CIM session,
+Windows PowerShell validates connection settings, and then establishes and
```
Comment (SteveL-MSFT) [inline reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md]: Unless something is "Windows PowerShell" specific, we should just use "PowerShell" (applies throughout the doc).

File: reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+version of the Windows operating system.
+
+In contrast, a cmdlet run by using a PSSession runs on the target
+computer. It requires Windows PowerShell on the target system.
+Furthermore, the cmdlet sends data back to the local computer.
+Windows PowerShell manages the data sent over the connection, and keeps
+the size within the limits set by Windows Remote Management (WinRM). CIM
+sessions do not impose the WinRM limits.
+
+PSSessions only work with WinRM. CimSessions can use DCOM.
```
Comment (SteveL-MSFT) [inline reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md]: WMI actually stands for Windows Management Instrumentation, although this statement isn't complete.  You can use CIM sessions against any CIM over WS-Man endpoint (Dell iDrac, HP iLO, are examples).

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md) | :warning:Warning | | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/4.0/Microsoft.PowerShell.Core/About/about_CimSession.md) | :warning:Warning | | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/5.0/Microsoft.PowerShell.Core/About/about_CimSession.md) | :warning:Warning | | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/5.1/Microsoft.PowerShell.Core/About/about_CimSession.md) | :warning:Warning | | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md)
[reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/4.0/Microsoft.PowerShell.Core/About/about_CimSession.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/5.0/Microsoft.PowerShell.Core/About/about_CimSession.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/5.1/Microsoft.PowerShell.Core/About/about_CimSession.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_CimSession.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutCimSession/reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C1%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201805011134175207-2373%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=A6yXzI%2BM0kskWAknLHD9W9U%2B3HUwpkuHRel0tcLj8UU%3D&st=2018-05-01T11%3A39%3A17Z&se=2018-06-01T11%3A44%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md (hunk 1 above)
Comment (sethvs) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_CimSession.md]: Fixed.

**Pull request merged**